lib/mix/lib/mix/tasks/new.ex

```diff
@@ -445,8 +445,8 @@ defmodule Mix.Tasks.New do
     use ExUnit.Case
     doctest <%= @mod %>
 
-    test "the truth" do
-      assert 1 + 1 == 2
+    test "salutates the world" do
+      assert <%= @mod %>.hello == :world
```

Comment:
@josevalim oh sorry, missed it, shame on me 🙈 I amended it in https://github.com/elixir-lang/elixir/pull/6159/commits/f1279dca091311b0cfc060917d6dc84ed91c3b6d